### ISSUE TITLE
what is trainval？

### ISSUE THREAD

**Issue by shoutOutYangJie** [wuhuikai/FastFCN#48]
Thanks for your code. 
You train your model on training and val dataset. Does it mean that the validation dataset also join in training dataset ? 
![image](https://user-images.githubusercontent.com/46734514/65402434-dcf63500-de00-11e9-8028-7bfd4948b5ad.png)

I don't understand the meaning of trainval. Waiting for your reply.

**Comment by wuhuikai** [comment]
Images and labels from both training set and val set are used for training the model, the result is then tested on the testing set.

**Comment by hitlhz666** [comment]
> 来自训练集和值集的图像和标签用于训练模型，然后在测试集上测试结果。

谢谢解答
